requirements.bazel.txt

```diff
@@ -38,3 +38,5 @@ typing-extensions==4.9.0
 pyasn1-modules==0.3.0
 zipp==3.17.0
 wrapt==1.16.0
+xds-protos==0.0.11
```

Comment:
`xds-protos` should be pulled in hermetically, not from `pypi`. It's available as `//tools/distrib/python/xds_protos`

Reply:
removed `xds-protos` no tests are breaking . i'll add it back in subsequent PR when it's required.